src/watchmaker/utils/__init__.py

```diff
@@ -83,3 +84,31 @@ def copytree(src, dst, force=False, **kwargs):
         shutil.rmtree(dst)
 
     shutil.copytree(src, dst, **kwargs)
+
+
+def config_none_deprecate(check_value, log):
+    r"""
+    Check whether a variable is the string 'None' rather than Pythonic `None`.
+
+    If it is the string 'None', this warns and returns Pythonic `None`.
+    Otherwise, the variable is returned unchanged.
+
+    Args:
+        check_value: (:obj:`str`)
+            Variable to be checked for string 'None' (case-insenstive).
+            (*Default*: None)
```

Comment:
I think there is no default value, both these params are required...

Reply:
Yes, absolutely. I misread. I was not thinking Pythonic `None` but rather that there was no default.